helm/mongodb/app-configs/mongodb_delivery.yaml

```diff
@@ -4,21 +4,19 @@ service:
 replicaCount: 3
 storageGi: 50Gi
 fsType: ext4
-pv0:  
-  region: eu-west-1
-  zone: eu-west-1a
-  name: mongodb-0-vol
-  volumeID: aws://eu-west-1a/vol-0f41bc74177761268
-  claimName: mongo-persistent-mongodb-0
-pv1:  
-  region: eu-west-1
-  zone: eu-west-1b
-  name: mongodb-1-vol
-  volumeID: aws://eu-west-1b/vol-0583d947a220fddb3
-  claimName: mongo-persistent-mongodb-1
-pv2:  
-  region: eu-west-1
-  zone: eu-west-1c
-  name: mongodb-2-vol
-  volumeID: aws://eu-west-1c/vol-063166f5c89299da4
-  claimName: mongo-persistent-mongodb-2
+pvs:
```

Comment:
We'll have to move these pvs to the upp-global-config. But I think we can do it later.